components/rollup_node/block_gen.md

```diff
@@ -0,0 +1,147 @@
+# Block Generation
+
+The logic which is used to generate the rollup chain from L1.
```

Comment:
What do you think about expanding this to clearly say that L1 is used as a data availability layer to generate the rollup chain and state?

Reply:
This is syncing from L1 from the PoV of a verifier/fresh sequencer - guessing the block generation from the PoV of the sequencer would go in the batch submitter doc?

Reply:
Great callout. I think it would be worth explicitly calling out the data availability layer

Reply:
Hmm @tynes we're going to add another doc which introduces block gen & feeds & whatnot. I think that will be the best place to add extra detail about availability layers. For now I just added the word "availability layer" but you're totally right that we should try to define what we need / mean by an availability layer because then it will become clearer to readers what we need to do to support eth2 shards